TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -278,12 +280,43 @@ internal void LoadCustomAnalyzers(ExtensionManager extensionManager)
             this._customAnalyzerProviders = extensionManager.Activate<IAnalyzerProvider>().ToArray();
         }
 
+        /// <summary>
+        /// Try to get the DocumentContext instance associated to an opened document from its Uri
+        /// </summary>
+        /// <param name="docUri">The Document's uri to get the associated DocumentContext instance.</param>
+        /// <param name="openedDocumentContext">[out] The DocumentContext instance of the opened document if any, null otherwise</param>
+        /// <returns>True if the uri was associated to an opened document, false otherwise</returns>
+        internal bool TryGetOpenedDocument(Uri docUri, out DocumentContext openedDocumentContext)
+        {
+            return this._allOpenedDocuments.TryGetValue(docUri, out openedDocumentContext);
+        }
+
+        /// <summary>
+        /// Try to get the DocumentContext instance by name
+        /// </summary>
+        /// <param name="name">The document's name</param>
+        /// <param name="openedDocumentContext"></param>
+        /// <returns>true if the DocumentContext has been found, false otherwise</returns>
+        internal bool TryGetOpenedDocumentByName(string name, out DocumentContext openedDocumentContext)
```

Comment:
This is a private method

Reply:
done in [50aad65](https://github.com/TypeCobolTeam/TypeCobol/pull/2158/commits/50aad651b088759acfc704aa689a84ee67050e2d)